webapp/geek_text/settings.py

```diff
@@ -122,3 +123,4 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+LOGIN_REDIRECT_ULR='/account/'
```

Comment:
This constant seems to be misspelled.
Please fix this; the end part of it is `ULR` instead of `URL`.

Reply:
Will do